src/datanode/src/instance.rs

```diff
@@ -240,7 +241,21 @@ pub(crate) async fn new_oss_object_store(store_config: &ObjectStoreConfig) -> Re
         config: store_config.clone(),
     })?;
 
-    Ok(ObjectStore::new(accessor))
+    let mut object_store = ObjectStore::new(accessor);
+    if let Some(path) = oss_config.cache_path.as_ref() {
+        let accessor =
+            FsBuilder::default()
+                .root(path)
+                .build()
+                .with_context(|_| error::InitBackendSnafu {
+                    config: store_config.clone(),
+                })?;
+        let cache_capacity = oss_config.cache_capacity.unwrap();
```

Comment:
We can assume that the user always set `cache_capacity` when setting `cache_path`, I think `unwrap_or(DEFAULT_OBJECT_STORE_CACHE_SIZE)` is better. Define constant `DEFAULT_OBJECT_STORE_CACHE_SIZE`.